src/ddspdrum/module.py

```diff
@@ -58,6 +62,94 @@ def control_to_sample_rate(self, control: np.array) -> np.array:
             )
             return resample(control, num_samples)
 
+    def add_parameter(self, parameter_id: str, value: float, minimum: float,
+                      maximum: float, scale: float = 1):
+        """
+        Add a new parameter to this module.
+
+        Parameters
+        ----------
+        parameter_id (str)  :   id to save this parameter as
+        value (float)       :   initial value for this parameter
+        minimum (float)     :   minimum value that this parameter can take
+        maximum (float)     :   maximum value that this parameter can take
+        scale (float)       :   scaling when converting between [0,1] range
```

Comment:
Not sure I completely understand this variable. I'll read on in the code, but I suspect this comment could be a little clearer.

Reply:
Agreed that I find this a bit confusing too